packages/webapp/public/locales/ml/common.json

```diff
@@ -1,6 +1,6 @@
 {
   "ABANDON": "ഉപേക്ഷിക്കുക",
-  "ACTIVE": "സജീവമാണ്",
+  "ACTIVE": "\nസജീവമാണ്",
```

Comment:
I missed what happened here.

Reply:
Um, it looks like it was entered with an extra carriage return, is that what you mean? I usually remove those on CrowdIn when I spot them 😂